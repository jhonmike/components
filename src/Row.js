import React from 'react'
import classnames from 'classnames'
import PropTypes from 'prop-types'

import styles from './row.css'

const Row = props => {
  const {
    className,
    children,
    ...otherProps
  } = props

  const classes = classnames(
    styles.row,
    className
  )

  return (
    <div {...otherProps} className={classes}>
      {children}
    </div>
  )
}

Row.propTypes = {
  className: PropTypes.string,
  children: PropTypes.node
}

export default Row
