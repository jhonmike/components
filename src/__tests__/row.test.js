import React from 'react'
import renderer from 'react-test-renderer'
import Row from '../Row'

describe('Row Component', () => {
  it('renders component', () => {
    const tree = renderer
      .create(<Row>Row</Row>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
