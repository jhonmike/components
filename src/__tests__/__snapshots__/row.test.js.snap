// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row Component renders component 1`] = `
<div
  className="row"
>
  Row
</div>
`;
